### PR TITLE
Added travis checks

### PR DESCRIPTION
**Pull request by carl-alberto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 WordPress Plugin Template
 =========================
 
+[![Build Status](https://travis-ci.org/carl-alberto/WordPress-Plugin-Template.svg?branch=master)](https://travis-ci.org/carl-alberto/WordPress-Plugin-Template)
```
Comment (grappler) [inline README.md]: You seem to have added your own travis link here.

File: codesniffer.ruleset.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Theme Coding Standards">
+	<!-- See https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml -->
+	<!-- See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml -->
+
+	<!-- Set a description for this ruleset. -->
+	<description>A custom set of code standard rules to check for WordPress themes.</description>
```
Comment (grappler) [inline codesniffer.ruleset.xml]: You may want to customize this more or at least change theme to plugin. I would also give credit where it is due. This seems to be taken from `_s`

File: README.md (hunk 1 above)
Comment (carl-alberto) [inline README.md]: Thanks for pointing that our, still new to WPCS and Travis :), updated my PR

File: codesniffer.ruleset.xml (hunk 2 above)
Comment (carl-alberto) [inline codesniffer.ruleset.xml]: Thanks for pointing that our, still new to WPCS and Travis :), updated my PR

Comment (grappler) [inline codesniffer.ruleset.xml]: Could you also mention where you copied the files from as that is normally a requirement of open source licensing?

Comment (carl-alberto) [inline codesniffer.ruleset.xml]: Sure, updated PR to reflect the source of the ruleset. Thanks for the guidance!

**Comment by jonathanbossenger** [comment]
@grappler are you happy with the latest updates to this?

**Comment by grappler** [comment]
@jonathanbossenger A lot has changed in the last years. I am not sure if the PHPCS is still working correctly in this travis file.

A few updates that could be made are:
- Replace `hhvm` with `nightly` as `hhvm` is not being used as activity anymore due to PHP 7.
- Test PHP 7.2
- Use composer to install the packages instead of git https://github.com/Automattic/_s/pull/1291

**Comment by jonathanbossenger** [comment]
Thanks. @carl-alberto if you have some free time, perhaps you can look at implementing these changes? Other wise I will try to do so over the course of the next few weeks.

**Comment by carl-alberto** [comment]
@jonathanbossenger thanks for the bump, I'll try to revisit this this weekend :)

**Comment by jonathanbossenger** [comment]
Thanks @carl-alberto no rush if you don't.

**Comment by carl-alberto** [comment]
@jonathanbossenger My branch carl-alberto:feature/wpcs-compliant is way outdated, when doing PR's do we still branch from master or develop?

**Comment by jonathanbossenger** [comment]
@carl-alberto thanks for asking, please submit all PR's against `develop`.

**Comment by jonathanbossenger** [comment]
Closing this because this was all completed with #75

**Pull request closed without merging**